### DISCUSSION TITLE
How to open an url on Datagrid cell click

### DISCUSSION THREAD

**Discussion by geronimogoemon** [perspective-dev/perspective#2017] [Q&A] [answered]
## Support Question

Hello!

I would like to open an URL in a new tab when user clicks one of the rows in the Datagrid. The url would be different depending on the data stored inside the row. I can't find anything similar in the [docs](https://perspective.finos.org/docs/js) or among github issues.

Is there some kind of callback for such purpose?

**Comment by texodus** [comment, marked as answer]
This is covered in the docs section deceptively-titled [Click Events](https://perspective.finos.org/docs/js#click-events).  There are accompanying examples docs, and at a quick glance the first 5 examples in the `README.md` all implement variations on this feature in a variety of ways.

Not covered in the docs (but not exactly a hidden feature) is that this is also trivially easy to configure in the GUI itself.

https://user-images.githubusercontent.com/60666/204036968-ff498738-1937-4590-a3e7-0c694efab57f.mov

**Comment by geronimogoemon** [comment]
Thank you so much! It was right at the bottom of the docs and that's probably why I missed it :D
